buildflow/samples/sqs_walkthrough.py

```diff
@@ -0,0 +1,33 @@
+# flake8: noqa
```

Comment:
nit: Maybe add a doc string on how to login to let buildflow access aws creds

Reply:
Done.